project/plugins.sbt

```diff
@@ -1,8 +1,8 @@
 // The Play plugin
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.17")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.11")
 
-addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.9")
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.18")
 
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.3")
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.9.0")
 
-addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.9.0")
\ No newline at end of file
+libraryDependencies += "org.vafer" % "jdeb" % "1.10" artifacts (Artifact("jdeb", "jar", "jar"))
```

Comment:
Could you explain the significance of this change and why it's necessary?

Reply:
I think we needed it at some point when we weren't going the whole hog with Scala 2.13. I've removed it and all seems OK – thanks for spotting!